client-docs/README.md

```diff
@@ -323,6 +340,14 @@ Accepts the following `opts`:
 - `role`: the role to grant for the invited device
 - `timeout`: the maximum amount of time in seconds to wait for a response.
 
+<!-- Should this return something? -->
```

Comment:
We discussed this a bit the other day. It would be nice if it resolved into some sort of success indication. Eg: `sucessfullyCancelled:boolean`. But not 100% necessary

Reply:
If possible, I think we should start with this returning `Promise<void>` and change it to `Promise<boolean>` (or something) as needed.